apps/documents/views.py

```diff
@@ -51,10 +51,11 @@ class DocumentViewSet(mixins.CreateModelMixin,
 
     def get_queryset(self):
         queryset = self.queryset
-        if settings.PROFILES_ENABLED:
-            shipment_id = self.kwargs['shipment_pk']
-            return queryset.filter(shipment__id=shipment_id)
-        return queryset.filter(owner_id=get_owner_id(self.request))
+        shipment_id = self.kwargs['shipment_pk']
+        queryset = queryset.filter(shipment__id=shipment_id)
+        # if settings.PROFILES_ENABLED:
```

Comment:
Remove commented code

Reply:
We also further simplify this method by getting rid of the `shipment_id` variable since it's used just once.